### DISCUSSION TITLE
install lua-language-server always fialed, the mason log in this

### DISCUSSION THREAD

**Discussion by ArcturusVirgo** [mason-org/mason.nvim#375] [Q&A] [answered]
[INFO  2022/9/3 16:58:18] ...acker\start\mason.nvim/lua/mason-core/installer/init.lua:110: Executing installer for Package(name=lua-language-server)
[ERROR 2022/9/3 16:58:19] ...acker\start\mason.nvim/lua/mason-core/installer/init.lua:146: Installation failed for Package(name=lua-language-server) error="...acker\\start\\mason.nvim/lua/mason-core/installer/init.lua:113: ...start\\mason.nvim/lua/mason-core/managers/github/init.lua:55: Failed to fetch latest release from GitHub API. Refer to :h mason-errors-github-api for more information."

How to solve this error, thanks

**Comment by williamboman** [comment, marked as answer]
> Refer to :h mason-errors-github-api for more information.

**Comment by beauwilliams** [comment]
@Zavijah did you find a solution to this problem?

**Reply by skewled** [reply]
From nvim you look at the message provided by "h mason-errors-github-api". This will tell you it's due to rate limits and to attempt using the GitHub cli 'brew install gh' and then authenticate with 'gh auth login'. (This is on a Mac. You can probably sub for windows/linux as needed)

**Reply by beauwilliams** [reply]
This did not fix it for me. But good place start.

**Reply by williamboman** [reply]
You might be running into SSL issues since the introduction of the [API service](https://github.com/williamboman/mason-registry-api), see https://github.com/williamboman/mason.nvim/issues/566 and https://github.com/williamboman/mason.nvim/issues/582 (`api.mason-registry.dev` is the domain in question)
